test/controllers/register/trustees/individual/TrusteePhoneControllerSpec.scala

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.register.trustees.individual
+
+import controllers.ControllerSpecBase
+import controllers.actions._
+import forms.PhoneFormProvider
+import identifiers.register.trustees.individual.TrusteeNameId
+import models.person.PersonName
+import models.{Index, NormalMode}
+import org.scalatest.BeforeAndAfterEach
+import org.scalatest.mockito.MockitoSugar
+import play.api.data.Form
+import play.api.mvc.Call
+import play.api.test.Helpers._
+import services.FakeUserAnswersService
+import utils.{FakeNavigator, UserAnswers}
+import viewmodels.{CommonFormWithHintViewModel, Message}
+import views.html.phoneNumber
+
+class TrusteePhoneControllerSpec extends ControllerSpecBase with MockitoSugar with BeforeAndAfterEach {
```

Comment:
extending MockitoSugar and BeforeAndAfterEach is not required here

Reply:
done